packages/canner/test/hocs/validation.test.js

```diff
@@ -21,7 +21,7 @@ describe('withValidation', () => {
       onDeploy,
       removeOnDeploy
     }
-    WrapperComponent = withValidationn(MockComponent);
+    WrapperComponent = withValidation(MockComponent);
   });
 
   it('should error state = false', () => {
```

Comment:
We can add the `async` keyword
```suggestion
  it('should error state = false',  async () => {
```